packages/zosjobs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
 
+## Recent Changes 
+- Enhancement: Added new command, `zowe jobs modify job [jobname] [jobid]` with options `--jobclass` and `--hold-status`. Can now
```

Comment:
In this directory, instead of the phrase "Added new command", should it be something like "Added new functions to support ... whatever" ?